app/models/reports/repository.rb

```diff
@@ -209,7 +256,7 @@ def denominator(region, period)
     #     region_2_slug: { period_1: value, period_2: value }
     #   }
     #
-    def cached_query(calculation, &block)
+    def region_period_cached_query(calculation, &block)
```

Comment:
Have we added other `_cached_queries`? Wondering why we're being more specific here. Is `cached_query` not descriptive enough?